src/data/shared/case-studies/case-studies-list.js

```diff
@@ -56,31 +67,66 @@ const caseStudiesData = {
     {
       title: 'Netflix',
       logo: Netflix,
-      description: '<strong>Netflix</strong> uses eBPF at scale for network insights.',
+      description: '<strong>Netflix</strong> uses eBPF at scale for network insights',
       links: [
         {
           text: 'Blog',
           to: 'https://netflixtechblog.com/how-netflix-uses-ebpf-flow-logs-at-scale-for-network-insight-e3ea997dca96',
         },
+        {
+          text: 'Video',
+          to: 'https://www.youtube.com/watch?v=16slh29iN1g',
+        },
+      ],
+    },
+    {
+      title: 'Microsoft',
+      logo: Microsoft,
+      description:
+        '<strong>Microsot</strong> uses eBPF to enhance the observability and inspection of processes within Kubernetes',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://www.youtube.com/watch?v=ilcYXPDSgu8&list=PLj6h78yzYM2Pm5nF_GmNQHMyt9CUZr2uQ&index=5',
+        },
+        {
+          text: 'Talk 2',
+          to: 'https://www.youtube.com/watch?v=tdcHnbLJClc&list=PLj6h78yzYM2Pm5nF_GmNQHMyt9CUZr2uQ&index=6',
+        },
       ],
     },
     {
       title: 'Android',
       logo: Android,
       description:
-        '<strong>Android</strong> uses eBPF to monitor network usage, power, and memory profiling.',
+        '<strong>Android</strong> uses eBPF to monitor network usage, power, and memory profiling',
       links: [
         {
           text: 'Docs',
           to: 'https://source.android.com/docs/core/architecture/kernel/bpf',
         },
+        {
+          text: 'Docs 2',
+          to: 'https://source.android.com/docs/core/data/ebpf-traffic-monitor',
+        },
+      ],
+    },
+    {
+      title: 'Samsung',
+      logo: Samsung,
+      description: '<strong>Samsung</strong> uses eBPF in their android devices for networking',
+      links: [
+        {
+          text: 'Talk',
+          to: 'https://www.youtube.com/live/EViAho-6qoc?t=5220&si=shsfjYQ5X5lnL5cL',
```

Comment:
Can you link to the actual talk rather than the stream?

Reply:
The actual standalone talk isn't listed on the youtube page yet. So I just used that pending when it becomes available